driver-core/src/main/com/mongodb/client/model/MongoTimeUnit.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.client.model;
+
+import com.mongodb.annotations.Beta;
+
+/**
+ * Units for specifying time-based bounds for {@linkplain Window windows} and output units for some time-based
+ * {@linkplain WindowedComputation windowed computations}.
+ *
+ * @since 4.3
+ */
+@Beta
+public enum MongoTimeUnit {
+    /**
+     * @mongodb.server.release 5.0
```

Comment:
Can we just move these annotations to the class level?

Reply:
Done.